booster/library/Booster/Syntax/ParsedKore/Internalise.hs

```diff
@@ -937,10 +968,10 @@ internaliseRewriteRule partialDefinition exs aliasName aliasArgs right axAttribu
                 }
     return
         RewriteRule
-            { lhs = lhs.term
-            , rhs = rhs.term
-            , requires = lhs.constraints
-            , ensures = rhs.constraints
+            { lhs
+            , rhs
+            , requires = mempty
```

Comment:
I think we are somehow losing the `requires` clause on the track here ... but this whole function should be dead code anyway so we'd rather remove the `alias` business altogether than fix this.